core/src/main/resources/hudson/logging/Messages.properties

```diff
@@ -21,4 +21,4 @@
 # THE SOFTWARE.
 
 LogRecorderManager.init=Initialing log recorders
-LogRecorderManager.DisplayName=log
\ No newline at end of file
+LogRecorderManager.DisplayName=System Log
```

Comment:
Can you add a new line at eof to make tooling happy?

Reply:
```suggestion
LogRecorderManager.DisplayName=System Log

```

Reply:
My tooling is quite happy as is. As is Jenkins', this file has never had a trailing newline.

Reply:
Git/GitHub don't really like files w/o them.